config/genesis.toml

```diff
@@ -4,12 +4,14 @@ timestamp = 1672531200
 
 [[accounts]]
 type = "BasicWallet"
+storage_mode = "off-chain"
 init_seed = "0xa123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
 auth_scheme = "RpoFalcon512"
 auth_seed = "0xb123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
 
 [[accounts]]
 type = "BasicFungibleFaucet"
+storage_mode = "off-chain"
```

Comment:
Should we default the faucet to `on-chain`?

Reply:
I don't see why not. Done!